azext_iot/dps/common.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+shared: Define shared data types(enums) and constant strings.
+"""
+
+
+# Parameter Arg Groups
+DPS_IDENTIFIER = "Device Provisioning Service Identifier"
```

Comment:
Maybe we can slightly shorten this across the board to `Provisioning Service Identifier` or similar since its currently very long
![image](https://user-images.githubusercontent.com/28658112/172503618-69e5def5-d1b9-46c0-ad28-5fa5ac834683.png)

Reply:
Going to use DPS cause it feels less awk and should be easily understood as device provisioning service from the help